docs/cookbooks/insights/convissue.md

```diff
@@ -18,7 +18,7 @@ This is a time-series view of the priority issues identified in your bot-user co
 
 | High-level metrics tracked on overview tab|
 | ---- |
-| **Messages with issues**: This represents the percentage of messages where the above issues have been identified out of the total messages exchanged between the user and bot. |
+| **Messages with issues**: This represents the percentage of messages where the below listed issues have been identified out of the total messages exchanged between the user and bot. |
```

Comment:
I see that we show the issues in the screenshot, can we clarify them here? Instead of the **below listed** we can say in (issues shown in the image) or whatever you think suits the best. 

(I referred it with the help doc, correct me if I am wrong)